### DISCUSSION TITLE
How can i make morph realtion searchable

### DISCUSSION THREAD

**Discussion by rohit-sukanfood** [rappasoft/laravel-livewire-tables#980] [General]
I m trying to implement morph relationship searchable but its not working properly 
` public function supplier(): MorphOne
    {
        return $this->morphOne(
            SupplierProxy::modelClass(),
            "account"
        );
    }`

so when i try to make it searchable its not working and i know this package does not support morph relationship query but is there any ways to do it.

**Comment by lrljoe** [comment]
Adjust your builder query, for example:
```
    /**
     * @return Builder
     */
    public function builder(): Builder
    {
        return $this->model::whereHasMorph(
            'supplier',
            [Retailer::class, Company::class],
            function (Builder $query) {
                $query->where('name', 'like', '%CompanyNameHere%');
            });
                
    }
```
